ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml

```diff
@@ -152,7 +152,7 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
     workflow: ipi-aws-ovn
 - as: ocp-image-ecosystem-aws-arm64
-  interval: 24h
+  run_if_changed: ^(test/extended/image_ecosystem)|^(examples/db-templates)|^(examples/quickstarts)|^(examples/image-streams)|^(examples/jenkins)|^(vendor/github.com/openshift/source-to-image)|^(vendor/github.com/openshift/imagebuilder)
```

Comment:
So this change changed the type of the job from a periodic (indicated by `interval`) to a *presubmit*. But that does not make any sense, because the `openshift/multiarch` repository does not actually exist, so it cannot have presubmits, and it definitely cannot have presubmits that only run if some content of the (nonexistent) repository changes.